localized/en-us/games/wordwindow/wordwindow_level.tsv

```diff
@@ -146,12 +146,12 @@ en-US	11	2	4	1	"एक बस में ७ बच्चे सवार है
-en-US	11	3	3	1	"दराज में ७ पेंसिल हैं। दान ने दराज में ९ और पेंसिल रखीं। दराज में कितने पेंसिल हैं?"	ÎÚæÁ ×ð¢ 7 Âð¢çâÜ ãñ¢Ð ÎæÙ Ùð ÎÚæÁ ×ð¢ 9 ¥æñÚ Âð¢çâÜ Ú¹è¢Ð ÎÚæÁ ×ð¢ ç·UUUUÌÙð Âð¢çâÜ ãñ¢?	14	16	18	20	pencils	pencils	pencils	pencils	7 + 9 = <16>	2	7	9	0	0					wordwindow-stone-red.png	wordwindow-stone-green.png		
+en-US	11	3	3	1	"दराज में ७ पेंसिल हैं। डान ने दराज में ९ और पेंसिल रखीं। दराज में कितने पेंसिल हैं?"	ÎÚUæÁ ×ð´ 7 Âð´çâÜ ãñ´Ð ÇæÙ Ùð ÎÚUæÁ ×ð´ 9 ¥õÚU Âð´çâÜ ÚU¹è´Ð ÎÚUæÁ ×ð´ ç·¤ÌÙð Âð´çâÜ ãñ´?	14	16	18	20	pencils	pencils	pencils	pencils	7 + 9 = <16>	2	7	9	0	0					wordwindow-stone-red.png	wordwindow-stone-green.png		
 en-US	11	3	4	1	"एक किसान ने लंच से पहले मकई की ५ पंक्तियाँ लगाईं। उसने लंच के बाद ७ और पंक्तियाँ लगाईं। उसने दिन के दौरान मकई की कितनी पंक्तियाँ लगाईं?"	°·UUUU ç·UUUUâæÙ Ùð Ü¢¿ âð ÂãÜð ×·UUUU§ü ·UUUè 5 Â¢çQUUUUØæ¡ Ü»æ§ü¢Ð ©âÙð Ü¢¿ ·ðUUU ÕæÎ 7 ¥æñÚ Â¢çQUUUUØæ¡ Ü»æ§ü¢Ð ©âÙð çÎÙ ·ðUUU ÎõÚæÙ ×·UUUU§ü ·UUUè ç·UUUUÌÙè Â¢çQUUUUØæ¡ Ü»æ§ü¢?	11	12	13	14	rows	rows	rows	rows	5 + 7 = <12>	2	5	7	0	0					wordwindow-stone-blue.png	wordwindow-stone-green.png		
 en-US	11	3	5	1	"माया के पास ५ मार्बल हैं। जादेन के पास ८ मार्बल हैं। उनके पास कुल मिलाकर कितने मार्बल हैं?"	×æØæ ·ðUUU Âæâ 5 ×æÕüÜ ãñ¢Ð ÁæÎðÙ ·ðUUU Âæâ 8 ×æÕüÜ ãñ¢Ð ©Ù·ðUUU Âæâ ·éUUUÜ ç×Üæ·UUUUÚ ç·UUUUÌÙð ×æÕüÜ ãñ¢?	11	12	13	14	marbles	marbles	marbles	marbles	5 + 8 = <13>	3	5	8	0	0					wordwindow-stone-green.png	wordwindow-stone-red.png		
 en-US	12	1	1	4	"मारिन के पास ७ गेंदें हैं। एलेन के पास मारिन की तुलना में ६ अधिक गेंदें हैं। एलेन के पास कितनी गेंदें हैं?"	×æçÚÙ ·ðUUU Âæâ 7 »ð¢Îð¢ ãñ¢Ð °ÜðÙ ·ðUUU Âæâ ×æçÚÙ ·UUUè ÌéÜÙæ ×ð¢ 6 ¥çÏ·UUUU »ð¢Îð¢ ãñ¢Ð °ÜðÙ ·ðUUU Âæâ ç·UUUUÌÙè »ð¢Îð¢ ãñ¢?	10	11	12	13	balls	balls	balls	balls	7 + 6 = <13>	4	7	13	0	0					wordwindow-stone-blue.png	wordwindow-stone-green.png		
 en-US	12	1	2	4	"बॉब के पास ४ पेंसिल हैं और उसकी बहन के पास उससे ८ अधिक पेंसिल हैं। बॉब की बहन के पास कितनी पेंसिल हैं?"	ÕæòÕ ·ðUUU Âæâ 4 Âð¢çâÜ ãñ¢ ¥æñÚ ©â·UUUè ÕãÙ ·ðUUU Âæâ ©ââð 8 ¥çÏ·UUUU Âð¢çâÜ ãñ¢Ð ÕæòÕ ·UUUè ÕãÙ ·ðUUU Âæâ ç·UUUUÌÙè Âð¢çâÜ ãñ¢?	4	8	12	16	pencils	pencils	pencils	pencils	4 + 8 = <12>	3	4	12	0	0					wordwindow-stone-yellow.png	wordwindow-stone-red.png		
-en-US	12	1	3	4	"केली के मुर्गे ने कल ६ अंडे दिए। आज, उसके मुर्गे ने कल की तुलना में ५ और अंडे दिए। आज उसके मुर्गे ने कितने अंडे दिए?"	·ðUUUÜè ·ðUUU ×é»ðü Ùð ·UUUUÜ 6 ¥¢Çð çÎ°Ð ¥æÁ, ©â·ðUUU ×é»ðü Ùð ·UUUUÜ ·UUUè ÌéÜÙæ ×ð¢ 5 ¥æñÚ ¥¢Çð çÎ°Ð ¥æÁ ©â·ðUUU ×é»ðü Ùð ç·UUUUÌÙð ¥¢Çð çÎ°?	10	11	12	13	eggs	eggs	eggs	eggs	6 + 5 = <11>	2	6	11	0	0					wordwindow-stone-red.png	wordwindow-stone-green.png		
+en-US	12	1	3	4	"केली के मुर्गी ने कल ६ अंडे दिए। आज, उसके मुर्गी ने कल की तुलना में ५ और अंडे दिए। आज उसके मुर्गी ने कितने अंडे दिए?"	·Ô¤Üè ·Ô¤ ×é»èü Ùð ·¤Ü 6 ¥´Çð çÎ°Ð ¥æÁ, ©â·Ô¤ ×é»èü Ùð ·¤Ü ·¤è ÌéÜÙæ ×ð´ 5 ¥õÚU ¥´Çð çÎ°Ð ¥æÁ ©â·Ô¤ ×é»èü Ùð ç·¤ÌÙð ¥´Çð çÎ°?	10	11	12	13	eggs	eggs	eggs	eggs	6 + 5 = <11>	2	6	11	0	0					wordwindow-stone-red.png	wordwindow-stone-green.png		
```

Comment:
केली `की` मुर्गी
`उसकी` मुर्गी

Chanakya text has mistakes. Below text is incorrect.
अौर